dataland-api-key-manager/DockerfileTest

```diff
@@ -1,6 +1,11 @@
 ARG DATALAND_API_KEY_MANAGER_BASE_VERSION
-FROM ghcr.io/d-fine/dataland/dataland_api_key_manager_base:${DATALAND_API_KEY_MANAGER_BASE_VERSION}
+FROM ghcr.io/d-fine/dataland/dataland_api_key_manager_base:${DATALAND_API_KEY_MANAGER_BASE_VERSION} as builder
 
-CMD ./gradlew :dataland-api-key-manager:bootRun --args='--spring.profiles.active=ci' --no-daemon --stacktrace
+FROM eclipse-temurin:17.0.5_8-jre-alpine
```

Comment:
What exactly is the difference between the old approach and the new approach using a jacoco agent?

Reply:
Ok, as discussed.